### ISSUE TITLE
Used folder namespaces are not updated when running class:move

### ISSUE THREAD

**Issue by lumnn** [phpactor/phpactor#1171]
When moving folder with class:move, use lines, that reference whole folder are not getting updated.  

I'm not sure really how to describe it properly for everyone to understand, so I've prepared following example:

Class `App\Base\UsedHere` uses `App\Imported\Foo` and `App\Imported\Bar`.
For it to work, it has a line with `use App\Imported;` and then it references the classes by using namespace `Imported\Foo`.

After running `class:move src/Imported src/Other`, the use line doesn't get updated to `use App\Other`.

This example code can be found here: https://gitlab.com/lumnn/phpactor-class-move-namespace-issue

Steps to reproduce problem:

1. `git clone https://gitlab.com/lumnn/phpactor-class-move-namespace-issue.git`
2. `composer install`
3. `vendor/bin/phpunit`
4. `phpactor class:move src/Imported src/Other`
5. If there is warning about moving folders (I've submited PR in appropriate repo) just run `mv src/Imported src/Other` for now, to workarround this
6. `vendor/bin/phpunit`

After all those steps the file `src/Base/UsedHere.php` on the line 5 still contains line

`use App\Imported;`

while it should have

`use App\Other;`

Lines 15 and 16 has been updated correctly with new folder name.

**Comment by dantleech** [comment]
yes, this is a long standing issue (this code was the first thing written for Phpactor and hasn't really been modified since) I think. class move _mostly_ works, but it has some issues. @BladeMF has been doing some work on a new implementation.

**Comment by lumnn** [comment]
It's not as much of an issue for me currently, I just thought it would be worth to report it, as I couldn't find any related issues reported so far.

I've updated the repository to be publicly available.

**Comment by BladeMF** [comment]
Currently the Rename just renames a class (just the name, not the namespace, which a move changes), but the same code could be used later for class moving.
